new/LIT14518Diagram.xml

```diff
@@ -0,0 +1,87 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7122Diagram" xml:lang="en" type="work">
```

Comment:
I understand, that LIT14518Diagram is identical to LIT7122Diagram and will be deleted in another pull request, correct?
Otherwise, xml:id must be changed, because it must correspond to the title of the file.